sass/components/_header.scss

```diff
@@ -118,18 +132,22 @@
   @include govuk-width-container;
   @include govuk-responsive-margin(3, 'bottom');
   @include govuk-responsive-padding(3, 'top');
-  @include govuk-responsive-padding(3, 'bottom');
```

Comment:
moved this down into the location bar and feedback, so when feedback wraps in next line we have a nice bottom padding